Sources/Fuzzilli/FuzzIL/JsOperations.swift

```diff
@@ -453,6 +453,15 @@ class Return: JsOperation {
     }
 }
 
+class DifferentialHash: Operation {
```

Comment:
One thing I'm wondering about here is whether we expect this operation to be present in corpus samples or not. Is this operation just emitted whenever a fuzzer testcase is executed, then removed afterwards (e.g. during minimization), similar to the `Explore` operation for example, or will it persist in the samples and be part of the corpus?

Reply:
Removing it entirely from the corpus (but obviously not from samples producing differentials) seems reasonable. How would I handle interesting samples where the removal of `DifferentialHash` changes the coverage?

Reply:
👍  in that case please make it inherit from `InternalOperation`, then we'll assert that it doesn't get into the Corpus. I think we don't really need to worry about the coverage. The edges that this triggers will be marked as non-deterministically fairly quickly, at which point they will no longer trigger. Might be worth adding a comment that explains this at the place where the `DifferentialHash` operations are removed.